fizz-buzz-streams/src/test/java/com/pubsap/eng/streams/FizzBuzzAllTopologySpec.java

```diff
@@ -0,0 +1,192 @@
+package com.pubsap.eng.streams;
+
+import com.pubsap.eng.schema.*;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigFactory;
+import io.confluent.kafka.schemaregistry.client.rest.exceptions.RestClientException;
+import io.confluent.kafka.streams.serdes.avro.SpecificAvroSerde;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.streams.*;
+import org.apache.kafka.streams.kstream.TimeWindows;
+import io.confluent.kafka.schemaregistry.client.MockSchemaRegistryClient;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.*;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.apache.kafka.common.utils.Utils.mkEntry;
+import static org.apache.kafka.common.utils.Utils.mkMap;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class FizzBuzzAllTopologySpec {
```

Comment:
🙂Let's stick with Java convention and call it a Test (not a Spec) `FizzBuzzCompleteTopologyTest`